docs/ldml/tr35.md

```diff
@@ -3589,7 +3589,7 @@ To canonicalize a given _source_:
 4. Transform _source_ according to that rule
 5. loop (goto #3)
 
-## Processing LocaleIds
+### <a name="processing-localeids" href="#processing-localeids">Processing LocaleIds</a>
```

Comment:
This and some following headers were at the wrong level

Reply:
was the fix-tocs tool used to regenerate this or done manually?

the ToC is updated automatically before html is generated, but should be run locally also

Reply:
For Appendix C, both the TOC and the section heads were at the wrong level for some sections, so TOC generation would not fix it. Also the link target and href were missing for all of the Appendix XC subsections (though present in the TOC somehow).